course02/homework00/skeleton/vector.h

```diff
@@ -1,8 +1,45 @@
 #pragma once
 
+#include <array>
+
 class VectorInt
 {
-	// TODO
+	
+public:
+	int32_t size()
+	{
+		return mSize;
+	}
+
+	int32_t capacity()
+	{
+		return mData.size();
+	}
+
+	int32_t back()
+	{
+		if (mSize > 0)		
+			return mData[mSize-1];
```

Comment:
Interesting choice - if there is nothing, you dont `return` anything! I would hope you get a compiler warning for this at least... Please fix this, at the very least if you want want to check anything at runtime and throw, please add an assert